packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -6015,35 +6016,56 @@ export class WalletService implements IWalletService {
 
   oneInchGetTokens(req): Promise<any> {
     return new Promise((resolve, reject) => {
+
       const credentials = this.oneInchGetCredentials();
+      const chain = req.params?.['chain'];
```

Comment:
Fallback to 'eth'? Actually, I would recommend falling back in the route handler since the chain param is optional (i assume for backwards compatibility from before polygon)